apps/src/code-studio/initApp/project.js

```diff
@@ -1868,12 +1872,12 @@ function fetchAbuseScoreAndPrivacyViolations(project) {
  */
 function setMakerAPIsStatusFromQueryParams() {
   if (hasQueryParam('enableMaker')) {
-    currentSources.makerAPIsEnabled = true;
+    currentSources.makerAPIsEnabled = 'circuitPlayground';
```

Comment:
I think we should make these constants so we don't need to worry about mistyping strings.

Reply:
Good point.  If you have the chance to make that change, that would be great.